static/app/utils/events.tsx

```diff
@@ -115,9 +119,23 @@ export function getTitle(event: Event | BaseGroup, features: string[] = []) {
         };
       }
 
+      const displayTitleWithTreeLabel =
+        features.includes('grouping-title-ui') &&
```

Comment:
`getTitle` is being used in many other places and because of that we have to leave this check here too